sdks/python/apache_beam/utils/value_provider_test.py

```diff
@@ -25,32 +25,58 @@
 
 
 class ValueProviderTests(unittest.TestCase):
-  def test_static_value_provider(self):
+  def test_static_value_provider_keyword_argument(self):
     class UserOptions(PipelineOptions):
       @classmethod
       def _add_argparse_args(cls, parser):
         parser.add_value_provider_argument(
             '--vp_arg',
-            help='This flag is a value provider',
+            help='This keyword argument is a value provider',
             default='some value')
     options = UserOptions(['--vp_arg', 'abc'])
     self.assertTrue(isinstance(options.vp_arg, StaticValueProvider))
     self.assertTrue(options.vp_arg.is_accessible())
     self.assertEqual(options.vp_arg.get(), 'abc')
 
-  def test_runtime_value_provider(self):
+  def test_runtime_value_provider_keyword_arguent(self):
     class UserOptions(PipelineOptions):
       @classmethod
       def _add_argparse_args(cls, parser):
         parser.add_value_provider_argument(
             '--vp_arg',
-            help='This flag is a value provider')
+            help='This keyword argument is a value provider')
     options = UserOptions()
     self.assertTrue(isinstance(options.vp_arg, RuntimeValueProvider))
     self.assertFalse(options.vp_arg.is_accessible())
     with self.assertRaises(RuntimeError):
       options.vp_arg.get()
 
+  def test_static_value_provider_positional_argument(self):
+    class UserOptions(PipelineOptions):
+      @classmethod
+      def _add_argparse_args(cls, parser):
+        parser.add_value_provider_argument(
+            'vp_pos_arg',
+            help='This positional argument is a value provider',
+            default='some value')
+    options = UserOptions(['abc'])
+    self.assertTrue(isinstance(options.vp_pos_arg, StaticValueProvider))
+    self.assertTrue(options.vp_pos_arg.is_accessible())
+    self.assertEqual(options.vp_pos_arg.get(), 'abc')
+
+  def test_runtime_value_provider_positional_argument(self):
+    class UserOptions(PipelineOptions):
```

Comment:
Change `UserOptions` to a more unique name, they potentially cause name conflicts with other tests.